README.md

```diff
@@ -24,9 +24,12 @@ config :procore,
   client_secret: "PROCORE_CLIENT_SECRET"
   host: "PROCORE_HOST_URL"
   oauth_url: "#{PROCORE_HOST_URL}/oauth/token",
-  http_client: Procore.HttpClient
+  http_client: Procore.HttpClient,
+  default_version: "v1.0"
 ```
 
+`default_version` sets the rest or vapid API version you would like to be used when not specified in your request.
```

Comment:
What do you think about the following update? It gives guidance about what value to set.

`default_version` sets the default API version to use if none is specified
in the client request. Should be either "v1.0" (recommended) or "vapid" (legacy).

Reply:
I like that, that's definitely better 👍